internals/config/default.yml

```diff
@@ -0,0 +1,193 @@
+---
+name: "default name"
+slug: "default"
+
+settings:
+  mobile_phone_verification: false
+  subscriptions: false
+
+# configuration variables for the server app
+server:
+  # put the URL of openwisp-radius here
+  host: "http://127.0.0.1:8000"
+  uuid: organization_uuid
+  secret_key: organization_secret_key
+  timeout: 2 #request timeout period in seconds
+
+# configuration variables for the client
+# Path of all the assets should be defined relative to the organization's folder
+# in assets directory.
+client:
+
+  auto_login: True
+
+  # path of favicon
+  favicon: "favicon.png"
+
+  # path of the custom css file relative to organization's folder in
+  # assets directory.
+  css_path: "index.css"
+  languages:
+    - text: "english"
+      slug: "en"
+  default_language: "en"
+  components:
+    header:
+      logo:
+        url: "openwisp-logo-black.svg" # logo url
+        alternate_text: "openwisp"
+      links: []
+
+    footer:
+      links: []
+      secondary_text: false
+
+    registration_form:
+      input_fields:
+        phone_number:
+          type: "tel"
```

Comment:
can we please eliminate all `type` attributes of all inputs?
This thing is only adding overhead and increasing the size of configurations with no benefits, so I think we can get rid of it.

Reply:
Do I have to remove type attributes from the whole code or just from the `default.yml` file?

Reply:
we can hardcode the type attributes of form inputs in the HTML and remove any reference from them in default.yml, the YAML generator and everything else.

The same could apply for any other property which does not make sense to allow to be configurable, for example: why should we allow to change the type attribute of the remember me checkbox? If you find any other occurrence of the same concept let me know.